AdminCore.DatabaseMigration/db/migrations/V1.0__original_schema.sql

```diff
@@ -439,4 +439,28 @@ WITH (
 TABLESPACE pg_default;
 
 ALTER SEQUENCE entitled_holiday_entitled_holiday_id_seq
-    OWNED BY entitled_holiday.entitled_holiday_id;
\ No newline at end of file
+    OWNED BY entitled_holiday.entitled_holiday_id;
+
+				----------------------------------------------------------------------------------------
+
+/*
+                                   SCHEDULED JOB TABLE
+*/
+
+  ----------------------------------------------------------------------------------------
+CREATE SEQUENCE IF NOT EXISTS public.scheduled_job_scheduled_job_id_seq;
+CREATE TABLE IF NOT EXISTS public.scheduled_job
+(
+    scheduled_job_id integer NOT NULL DEFAULT nextval('scheduled_job_scheduled_job_id_seq'::regclass),
+    schedule_job_name text NOT NULL,
+		schedule_cron_expression text NOT NULL,
```

Comment:
just spacing here.

Reply:
Weird, this doesn't seem to show up in my local, I've closed it to work on changes since then so maybe it was fixed just after this PR was opened.